src/js/components/reviewData/ReviewDataComments.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * ReviewDataComments.jsx
+ * Created by Alisa Burdeyny 11/21/16
+ **/
+
+import React from 'react';
+import ReviewDataCommentsDropdown from './ReviewDataCommentsDropdown.jsx'
+import ReviewDataCommentsTextfield from './ReviewDataCommentsTextfield.jsx'
+
+import * as ReviewHelper from '../../helpers/reviewHelper.js';
+
+export default class ReviewDataComments extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentFile: "A",
+            fileComments: props.comments,
```

Comment:
Let's not mix our props and states. Additionally, in an async situation, the component may mount before the server responds with the server prop values.